contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -3222,3 +3222,375 @@ BEGIN
 END;
 $$
 LANGUAGE pltsql STABLE;
+
+-- sp_babelfish_autoformat is a helper procedure which formats the contents of a table (or view)
+-- as narrowly as possible given its actual column contents.
+-- This proc is currently only used by sp_who but could be applied more generically.
+-- A complication is that the metadata from #tmp tables cannot be found in the babelfish
+-- catalogs, so we have to use some trickery to make things work.
+-- Not all datatypes are handled as well as might be possible, but it is sufficient for 
+-- the current purposes.
+-- Note that this proc may increase the response time for the first execution of sp_who, but 
+-- we're looking at prioritizing user-friendliness (easy-to-read output) here. Also, sp_who 
+-- is very unlikely to be part of performance-critical workload.
+CREATE OR REPLACE PROCEDURE sys.sp_babelfish_autoformat(
+	IN "@tab"        VARCHAR(257) DEFAULT NULL,
+	IN "@orderby"    VARCHAR(1000) DEFAULT '',
+	IN "@printrc"    sys.bit DEFAULT 1,
+	IN "@hiddencols" VARCHAR(1000) DEFAULT NULL)
```

Comment:
VARCHAR and sys.VARCHAR are not used consistently in this source file. For example, sp_helpdb and and sp_getapplock (and others) also uses VARCHAR (without sys.) and I took these as guidance since these procedures have existed since v.1.0. However sys.VARCHAR is also used in other objects so doing some code cleanup would make sense but that is an action beyond the scope of this PR.

Reply:
Earlier we did not follow proper guidelines in creating and declaring objects, now with new objects or modifications to the current objects, We are making sure to use sys.VARCHAR instead of VARCHAR